Intersect.Client/Entities/Player.cs

```diff
@@ -105,6 +105,19 @@ public Guid Class
         Dictionary<Entity, TargetInfo> mlastTargetList = new Dictionary<Entity, TargetInfo>(); // Entity, Last Time Selected
 
         Entity mLastEntitySelected = null;
+        
+        private readonly Directions[][] mOppositePlayerDirections =
+        {
+            new[] { Directions.Down, Directions.DownLeft, Directions.DownRight },
+            new[] { Directions.Up, Directions.UpLeft, Directions.UpRight },
+            new[] { Directions.Right, Directions.UpRight, Directions.DownRight },
+            new[] { Directions.Left, Directions.UpLeft, Directions.DownLeft },
+            
+            new[] { Directions.DownRight, Directions.Right },
+            new[] { Directions.DownLeft, Directions.Left },
+            new[] { Directions.UpLeft, Directions.Left },
+            new[] { Directions.UpRight, Directions.Right },
+        };
```

Comment:
1. No

2. What was this even for?

Reply:
> 1. No
> 
>     2. What was this even for?

1. ? ...
2. I'll add comments, it is just used for comparing the direction towards target with the directions where the player isn't currently facing at and essentially limits this feature (auto turn to target) so players don't turn around to their target if they are i.e, out of the player's front view ; the dictionary is made when starting up the client (readonly) and doesn't change at all afterwards. - If any of this matters, i don't pesonally like to limit this feature either 😞, but some developers and users have expressed cornern towards it.

Reply:
replaced this with an easier to read way to handle it, also with comments included